package.json

```diff
@@ -41,6 +41,7 @@
     "sinon-as-promised": "^2.0.3",
     "sinon-chai": "^2.7.0",
     "supertest": "^0.15.0",
-    "xunit-file": "0.0.6"
+    "xunit-file": "0.0.6",
+    "nock": "^2.17.0"
   }
```

Comment:
nock (https://github.com/pgte/nock) is a famous HTTP mocking and expectations library, I plan to add it to the spec/helper.js, but by different PR, how do you think? @RackHD/corecommitters